packages/@okta/vuepress-site/docs/guides/oauth-for-okta-apis/scopes/index.md

```diff
@@ -0,0 +1,45 @@
+| `okta.idps.manage`       | Allows the app to create and manage Identity Providers in your Okta organization| [Identity Providers API](/docs/reference/api/idps/#getting-started)|
+| `okta.idps.read`         | Allows the app to read information about Identity Providers in your Okta organization| [Identity Providers API](/docs/reference/api/idps/#getting-started)|
+| `okta.inlineHooks.manage`| Allows the app to create and manage Inline Hooks in your Okta organization.| [Inline Hooks API](/docs/reference/api/inline-hooks/)|
+| `okta.inlineHooks.read` | Allows the app to read information about Inline Hooks in your Okta organization.| [Inline Hooks API](/docs/reference/api/inline-hooks/)|
+| `okta.logs.read`         | Allows the app to read information about System Log entries in your Okta organization| [System Log API](/docs/reference/api/system-log/)|
+| `okta.schemas.manage`    | Allows the app to create and manage Schemas in your Okta organization   | [Schemas API](/docs/reference/api/schemas/#getting-started)|
+| `okta.schemas.read`      | Allows the app to read information about Schemas in your Okta organization| [Schemas API](/docs/reference/api/schemas/#getting-started)|
+| `okta.users.manage`      | Allows an app to manage any user's profile and credential information and to create new users| [Users API](https://developer.okta.com/docs/reference/api/users/#user-operations), [User Lifecycle Operations](/docs/reference/api/users/#lifecycle-operations)|
+| `okta.users.read`        | Allows an app to read any user's profile and credential information      | [Users API](https://developer.okta.com/docs/reference/api/users/#user-operations), [User Lifecycle Operations](/docs/reference/api/users/#lifecycle-operations)|
+| `okta.users.manage.self` | Allows an app to manage the currently signed-in user's profile           |   |
+| `okta.users.read.self`   | Allows an app to read the currently signed-in user's profile and credential information|  |
+
+## Scope Hierarchy 
+The available scopes exist in a hierarchy, so that the `manage` scopes can do everything that the `read` scopes do, but more. Additionally, the `self` scopes only allow for access to the user who authorized the token. For example, a `GET` request to the `/users` endpoint with the `okta.users.read` scope returns all the users that the admin has access to. If the same request is sent with the `okta.users.read.self` scope, only the current user's account returns.
+
+## Silent Downscoping
+The Okta Org Authorization Server returns all scopes that you request as long as the client app is permitted to request that scope (granted to the client app). It doesn't matter whether you have permissions for all the scopes that you request. If the scopes requested exist in the app's grants collection, those scopes are sent back in the access token. However, when you make a request to perform an action that you don't have permission to perform, the token doesn't work and you receive an error.
```

Comment:
'permitted to use that scope'? An app is permitted to *request* any scope. It just can't receive any scope.